packages/core/tsdown.config.ts

```diff
@@ -2,6 +2,10 @@ import { defineConfig } from 'tsdown'
 
 export default defineConfig({
   entry: ['src/index.ts'],
+  format: ['cjs', 'esm'],
```

Comment:
This feels like it shouldn't be an option at this point, but for adoption it's probably cleaner to have right now.

Reply:
VS Code only recently started supporting esm in _some_ extension hosts. I mean... https://github.com/microsoft/vscode/issues/130367